pkg/wallet/wallet.go

```diff
@@ -75,6 +75,13 @@ func NewWalletFromFile(path string) (*Wallet, error) {
 	}
 	defer file.Close()
 
+	stat, err := file.Stat()
+	if err != nil {
+		return nil, fmt.Errorf("check wallet file: %w", err)
+	}
+	if stat.Size() == 0 {
+		return nil, fmt.Errorf("wallet file is empty, please initialize it first")
+	}
```

Comment:
I'm against of `Stat` usage, and I don't think we need this code at all. The behaviour described in the target issue is outdated, what is the NeoGo version you're getting the described error on? The current behaviour is:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ ./bin/neo-go wallet import --wif qweqweqweqwe -w wallet.json
open wallet: open wallet.json: no such file or directory
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ touch wallet.json
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ ./bin/neo-go wallet import --wif qweqweqweqwe -w wallet.json
unmarshal wallet: EOF
```
The second error is more detailed than presented in the issue. We can make it even more detailed and similar to the first error:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ git diff
diff --git a/pkg/wallet/wallet.go b/pkg/wallet/wallet.go
index b509fd08..ee4b6887 100644
--- a/pkg/wallet/wallet.go
+++ b/pkg/wallet/wallet.go
@@ -71,7 +71,7 @@ func NewInMemoryWallet() *Wallet {
 func NewWalletFromFile(path string) (*Wallet, error) {
        file, err := os.Open(path)
        if err != nil {
-               return nil, fmt.Errorf("open wallet: %w", err)
+               return nil, fmt.Errorf("failed to open wallet: %w", err)
        }
        defer file.Close()
 
@@ -79,7 +79,7 @@ func NewWalletFromFile(path string) (*Wallet, error) {
                path: file.Name(),
        }
        if err := json.NewDecoder(file).Decode(wall); err != nil {
-               return nil, fmt.Errorf("unmarshal wallet: %w", err)
+               return nil, fmt.Errorf("failed to unmarshal wallet '%s': %w", path, err)
        }
        return wall, nil
 }
@@ -92,7 +92,7 @@ func NewWalletFromFile(path string) (*Wallet, error) {
 func NewWalletFromBytes(wallet []byte) (*Wallet, error) {
        wall := &Wallet{}
        if err := json.NewDecoder(bytes.NewReader(wallet)).Decode(wall); err != nil {
-               return nil, fmt.Errorf("unmarshal wallet: %w", err)
+               return nil, fmt.Errorf("failed to unmarshal wallet from bytes: %w", err)
        }
 
        return wall, nil
```